appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: 1.0.{build}
+
+platform: x64
+shallow_clone: true # Because the git clone takes ages due to its size
+
+# Cf. https://stefanscherer.github.io/setup-windows-docker-ci-appveyor/
+# Visual Studio 2017 allows to run Windows's Containers 
+image:
+- Visual Studio 2017
+
+# Show Windows and docker versions
+install:
+- ps: Write-Host Server version $(gp 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion').BuildLabEx
+- docker version
+
+build:
+  verbosity: minimal
+
+build_script:
+- ps: docker build -t traefik:windows ./windows/
+
+test_script:
+- ps: >-
+    docker run --name lb -d -p 8080:8080 traefik:windows --api --debug
+
+    docker ps
+
+    Invoke-WebRequest -Uri http://localhost:8080
```

Comment:
Oh that's nice. D4W does the magic to use localhost here. :-)
With just Docker EE installed you had to use the container IP instead.

Reply:
👍 I initially used your `curl` container for windows, but the download time was slowing the build. Tried this and it worked: D4W is already set up in Appveyor which is awesome.